lib/quantum/normalizer.ex

```diff
@@ -51,6 +51,18 @@ defmodule Quantum.Normalizer do
     normalize_options(base, %{schedule: schedule, task: task})
   end
 
+  def normalize(%Job{} = base, {schedule, task, opts}) when is_list(opts) do
+    if Keyword.keyword?(opts) do
```

Comment:
Why do we need this check?

Reply:
To avoid errors in case this config is passed
```
{"@daily", {Backup, :backup, []}, [1, 2, 3]}}
```
I would add this keyword list check as a guard if it was possible
I can remove it if you prefer to crash instead of ignoring

Reply:
@fly49 Silently ignoring the options if invalid does not seem like a good option to me. I'd rather have a crash instead.